src/pages/GamePage.jsx

```diff
@@ -84,42 +86,57 @@ const GamePage = () => {
   const resetAll = () => {
     setResultHandler(false);
     setTriesCounter(0);
-    setAlertContent((arr) => [
-      ...arr,
-      {
-        text: `Alert od zresetowania danych `,
-        location: true,
-      },
-    ]);
+    setAllAlertContent(addAlert(allAlertContent, "Zresetowano dane", true));
   };
 
-  const alertBoxArray = alertContent.map((elem, index) => {
+  const alertBoxArray = allAlertContent.map((elem, index) => {
     return {
       alertText: elem.text,
       mainArrayKey: index,
-      isBottom: elem.location,
+      isLocationBottom: elem.isLocationBottom,
     };
   });
 
+  const switchAlertPositionSpl = (mainIndex) => {
+    const switchSelectedItem = (arr) => {
+      return {
+        ...arr[mainIndex],
+        isLocationBottom: !arr[mainIndex].isLocationBottom,
+      };
+    };
+    allAlertContent.splice(mainIndex, 1, switchSelectedItem(allAlertContent));
+    setAllAlertContent(allAlertContent); // tu mam problem co przypisac do seta skoro splice modyfikuje tab
```

Comment:
nie powinienes modyfikowac splicem hoka. `useState` ma metode do modyfikacji wlasnie po to zeby jej uzywac